app/Exports/CreateCSVExport.php

```diff
@@ -660,14 +660,22 @@ public function query ()
                     ]);
             }
 
-            else
+            else if ($this->location_type === 'country')
             {
                 return Photo::with(['smoking', 'food', 'coffee', 'alcohol', 'softdrinks', 'other', 'sanitary', 'brands', 'dumping', 'industrial', 'art'])
                     ->where([
                         'country_id' => $this->location_id,
                         'verified' => 2
                     ]);
             }
+
+            else
+            {
+                return Photo::with(['smoking', 'food', 'coffee', 'alcohol', 'softdrinks', 'other', 'sanitary', 'brands', 'dumping', 'industrial', 'art'])
```

Comment:
Does this return all the photos? If we need this we should cluster it. I'm thinking this job is too big...

Reply:
Yes, this returns all the photos where verified = 2. What do you mean by cluster it? Indeed, the job is big, can we try it on the production server, without allowing the users to interact with it? We can never know for sure unless we try it.

Reply:
I meant chunking. Instead of getting all data, we can get it in chunks of 5,000 photos for example

Reply:
I'm removing this feature from this PR entirely, let's think about it sometime later.